emukit/core/optimization/gradient_acquisition_optimizer.py

```diff
@@ -67,3 +66,15 @@ def f_df(x):
 
         x_min, fx_min = min(optimized_points, key=lambda t: t[1])
         return x_min, -fx_min
+
+    def _get_anchor_points_generator(self, acquisition):
```

Comment:
As noted above, we may be able to get rid of this getter

Reply:
Now removed